### ISSUE TITLE
touchZoomRotate prevents both rotation and zoom functionality. Can we prevent rotation and allow zoom in-zoom out to users when they touche/pinch the screen?

### ISSUE THREAD

**Issue by anjaliD-syn** [mapbox/mapbox-gl-js#13046]
Hello team, 

Can we prevent rotation but still allow zoom in and zoom out in map for mobile devices where user can pinch/touch screen?
I tried setting up touchZoomRotate as false but it prevents both rotation and zooming in map using touch gesture. 
We want to disable the rotation only and allow user to zoom in and zoom out in the map.

**Comment by rithik56** [comment]
`map.on("load", () => {
  map.touchZoomRotate.enable();
  map.touchZoomRotate.disableRotation();
});
`
@anjaliD-syn @stepankuzmin I believe this would solve the above problem

**Comment by rithik56** [comment]
Please let me know if this doesn't solve the above issue

**Comment by hntddt123** [comment]
It's now    
map.target.touchZoomRotate.enable();
map.target.touchZoomRotate.disableRotation();

**Comment by Tusharkanta407** [comment]
Hello , This issue still exist ?

**Comment by ibesora** [comment]
No, rotation can be disabled via [disableRotation](https://docs.mapbox.com/mapbox-gl-js/api/handlers/#touchzoomrotatehandler#disablerotation)
